tsl/test/shared/expected/dist_distinct.out

```diff
@@ -888,6 +888,34 @@ LIMIT 10;
                            Remote SQL: SELECT "time" FROM public.metrics_dist WHERE _timescaledb_internal.chunks_in(public.metrics_dist.*, ARRAY[1, 2, 3]) ORDER BY public.time_bucket('01:00:00'::interval, "time") ASC NULLS LAST
 (23 rows)
 
+SELECT DISTINCT without any var references is handled correctly
+EXPLAIN (verbose, costs off)
+SELECT DISTINCT 1, 'constx'
+FROM metrics_dist;
+                                                                    QUERY PLAN                                                                     
+---------------------------------------------------------------------------------------------------------------------------------------------------
+ Unique
+   Output: 1, 'constx'::text
+   ->  Custom Scan (AsyncAppend)
+         Output: 1, 'constx'::text
+         ->  Append
+               ->  Custom Scan (DataNodeScan) on public.metrics_dist metrics_dist_1
+                     Output: 1, 'constx'::text
+                     Data node: data_node_1
+                     Chunks: _dist_hyper_7_37_chunk, _dist_hyper_7_40_chunk, _dist_hyper_7_43_chunk
+                     Remote SQL: SELECT NULL FROM public.metrics_dist WHERE _timescaledb_internal.chunks_in(public.metrics_dist.*, ARRAY[1, 2, 3])
```

Comment:
I guess it is weird that we even do a query against data nodes in this case. But perhaps that is another fix or just a weirdness with the PostgreSQL query plan?

Reply:
Another fix in the fdw layer. PostgreSQL is flexible in terms of the myriad queries it allows. 

`SELECT DISTINCT const1 from table` 

is allowed and that's what we are trying to not crash for here. IMO, nothing else needed to optimize for such queries.

Reply:
Added the DISTINCT pushdown here now.